src/minuteman_lashup_vip_listener.erl

```diff
@@ -60,7 +60,7 @@
 -type ip_vip() :: {tcp, inet:ip4_address(), inet:port_number()}.
 -type vip_name() :: binary().
 -type named_vip() :: {tcp, {name, {vip_name(), framework_name()}}, inet:port_number()}.
--type vip() :: {ip_vip() | named_vip(), [ip_port()]}.
+-type vip() :: {ip_vip() | named_vip(), [{inet:ip4_address(), ip_port()}]}.
```

Comment:
yeah, that would have made sense if it was only for overlay. Even though the reason for the fix is overlay, it is used even for non-overlay vip. Just that in case of non-overlay vip the two ips are same.